compiler/rustc_mir_transform/src/coverage/spans.rs

```diff
@@ -393,27 +393,25 @@ impl<'a> CoverageSpansGenerator<'a> {
     }
 
     fn check_invoked_macro_name_span(&mut self) {
```

Comment:
Could you use the opportunity to add a comment explaining what this function does and why?

Reply:
I'm still not entirely sure why this function does what it does, but I've added a brief description of what it's doing.